### PR TITLE
Add price calculation

### PR DESCRIPTION
**Pull request by amityadav0**
- Add price calculation
- Remove allowance from ui
- Fix add liquidity bug

up

### CODE REVIEW

File: src/components/CreatePoolModal.js [hunk 1]
```diff
@@ -40,32 +41,41 @@ export default function CurrencySelectionModal() {
...
+        tokenAllowance: 0,
+        tokenAddress: input
+      }
     })
   }
 
   async function handleNearAmount(event) {
     let input = event.target.value;
     setNearAmount(input);
+    console.log(input, "NEAR");
```
Comment (robert-zaremba) [inline src/components/CreatePoolModal.js:62]: let's remove this log

File: src/components/CreatePoolModal.js [hunk 2]
```diff
@@ -40,32 +41,41 @@ export default function CurrencySelectionModal() {
...
+    handlePrices();
   }
 
-  function increaseAllowance() {
-    let token = {
-      address: inputs.state.createPoolModal.tokenAddress,
-      amount: tokenInputAmount
+  async function handlePrices() {
+    console.log(tokenInputAmount, nearAmount, "jasshdlkas");
```
Comment (robert-zaremba) [inline src/components/CreatePoolModal.js:67]: same here

**Review by robert-zaremba — APPROVED** [review]

File: src/components/CreatePoolModal.js (hunk 1 above)
Comment (amityadav0) [reply src/components/CreatePoolModal.js:62]: ok

File: src/components/CreatePoolModal.js (hunk 2 above)
Comment (amityadav0) [reply src/components/CreatePoolModal.js:67]: ok

**Pull request merged**